### DISCUSSION TITLE
Are the DNS queries stored in Redis cache deleted from database after expiring their timeout counting?

### DISCUSSION THREAD

**Discussion by alpominth** [0xERR0R/blocky#841] [General]
In [a documentation of Unbound](https://nlnetlabs.nl/documentation/unbound/unbound.conf/) there is this citation:

`"It should be  noted  that  Unbound  never  removes  data  stored in the Redis server, even if some data have expired in terms of DNS TTL or the Redis  server  has  cached too  much  data;  if  necessary  the Redis server must be configured to limit the cache size, preferably with some kind of  least-recently-used eviction  policy."`

Does the same thing applies to Blocky? Or does it delete automatically the expired DNS entries?

**Comment by kwitsch** [comment]
This doesn't apply to blocky.

Redis entries are stored with the same TTL as the response TTL and will therefore automatically be removed.

**Comment by alpominth** [comment]
Great, I got.
